docs/rules/space-before-function-parentheses.md

```diff
@@ -1,5 +1,7 @@
 # Require or disallow spaces before function parentheses (space-before-function-parentheses)
 
+**Deprecation notice**: This rule is deprecated, as it has been renamed to [space-before-function-parens](space-before-function-parens.md) for consistency with other rules' names, which used "parens" instead of "parentheses". This rule, by the old name, will be removed in ESLint v1.0. The new rule is identical in everything except name.
+
```

Comment:
I think you should also update the deprecation warning of `space-after-function-name` which links to `space-before-function-parentheses`.